processor/src/main/java/org/mapstruct/ap/internal/model/source/SubclassMappingOptions.java

```diff
@@ -84,7 +92,7 @@ private static boolean isConsistent(SubclassMappingGem gem, ExecutableElement me
                         targetSubclass.toString() );
             isConsistent = false;
         }
-        subclassValidator.isInCorrectOrder( method, gem.mirror(), targetSubclass );
+        subclassValidator.isValidUsage( method, gem.mirror(), targetSubclass );
```

Comment:
We never use the return type from `isValidUsage`. Should we do that and assign it to `isConsistent` if it is `false`?

Reply:
it is used at another location:
https://github.com/mapstruct/mapstruct/pull/2708/files#diff-86e6eb61ca7c458ab2ea7274f6950ab596031e8a450771263d5c7492ed151862R198

Reply:
fixed this as specified using the `bitwise and` (`&` instead of `&&`) operator.